### PR TITLE
fix: import bug

### PR DESCRIPTION
**Pull request by Nico19941206**
- https://github.com/vesoft-inc/nebula-explorer-ent/issues/1770
- https://github.com/vesoft-inc/nebula-explorer-ent/issues/1658

### CODE REVIEW

File: app/utils/import.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { message } from 'antd';
 import _ from 'lodash';
-
```
Comment (hetao92) [inline app/utils/import.ts:3]: use getI18n, we dont use react-intl-universal directly now

File: app/pages/Import/TaskCreate/PasswordInputModal/index.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const PasswordInputModal = (props: IProps) => {
         onChange={e => setPassword(e.target.value)}
       />
       <div className={styles.btns}>
-        <Button onClick={() => handleConfirm()}>
```
Comment (hetao92) [inline app/pages/Import/TaskCreate/PasswordInputModal/index.tsx:36]: use handleCancel, it will not clear password if use onCancel

Comment (Nico19941206) [reply app/pages/Import/TaskCreate/PasswordInputModal/index.tsx:36]: this is props event

**Review by hetao92 — APPROVED** [review]
LGTM

**Pull request merged**